src/mmeGrpcClient/mmeGrpcClient.cpp

```diff
@@ -199,7 +199,8 @@ class MmeGrpcCliClient {
 	    cout << "total_num_of_subscribers                " << reply.total_num_of_subscribers() << endl;
 	    cout << "num_of_subscribers_detached             " << reply.num_of_subscribers_detached() << endl;
 	    cout << "num_of_tau_response_to_ue_sent          " << reply.num_of_tau_response_to_ue_sent() << endl;
-
+	    cout << "num_of_service_reject_sent	             " << reply.num_of_service_reject_sent() << endl;
```

Comment:
Output strings should not be variable name. Must be user friendly name

Reply:
Created jira task.